### ISSUE TITLE
VSCode displays only the first line of multiple after-member ///< doxygen comment lines

### ISSUE THREAD

**Issue by tobiasherzke** [microsoft/vscode#165726]
Does this issue occur when all extensions are disabled?: 
Not applicable: When started with --disable-extensions, VSCode does not display doxygen comments at all.

- VS Code Version: 1.73.0
- OS Version: Windows 10 Pro Version 21H2 x64 build 19044.2130

Steps to Reproduce:

1. Create new C++ file "main.cpp" with following content:
```
struct C {
  /// First line of doxygen for member var1, followed by the
  /// second line.
  int var1; 
  int var2; ///< First line of doxygen for member var2, followed by the
            ///< second line.
};
int main(int, char**) {
  struct C c;
  c.var1 = 0;
  c.var2 = 0;
  return c.var1 + c.var2;
}

```
3. Save the file, then hover the mouse cursor over occurrences of var1 or var2 in the editor window.
4. For var1, a tooltip will correctly display
> int C::var1
> First line of doxygen for member var1, followed by the
> second line.
5. But for var2, the tooltip only displays
> int C::var2
> /< First line of doxygen for member var2, followed by the

According to the doxygen documentation at https://www.doxygen.nl/manual/docblocks.html#memberdoc, multiple lines of doxygen comments starting with `///<` can follow a member definition. And in fact, when generating documentation with the doxygen command, both lines from the example file are included in the doxygen-generated documentation of var2.

**Comment by alexr00** [comment]
By default VS Code doesn't show those hovers. They should be coming from one of the extensions you have installed.

**Comment by VSCodeTriageBot** [comment]
This issue is caused by an extension, please file it with the repository (or contact) the extension has linked in its overview in VS Code or the [marketplace](https://aka.ms/vscodemarketplace) for VS Code. See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting). If you don't know which extension is causing the problem, you can run `Help: Start extension bisect` from the command palette (F1) to help identify the problem extension.

Happy Coding!

**Comment by tobiasherzke** [comment]
Thanks for the pointer. By searching in this direction, I've found:

[This announcement](https://devblogs.microsoft.com/cppblog/visual-studio-code-c-extension-july-2020-update-doxygen-comments-and-logpoints/) suggests that the Visual Studio Code C++ extension is responsible.

Related behavior of the extension (which however does not actually describe an error IMO) was [posted as an issue here](https://github.com/microsoft/vscode-cpptools/issues/5811). This extension issue report then triggered opening [this issue for Visual Studio](https://developercommunity.visualstudio.com/t/issues-with-multi-line-c-doxygen-doc-comments-afte/1124109) which includes the error case described in my post. The Visual Studio issue was then closed without fixing because of low priority.
